src/main/java/gof/strategy/services/strategy/CampaignBannerUpdateStrategy.java

```diff
@@ -0,0 +1,52 @@
+package gof.strategy.services.strategy;
+
+import gof.strategy.domain.EventResourceType;
+import gof.strategy.domain.EventType;
+import gof.strategy.domain.ScheduledEvent;
+import gof.strategy.domain.Status;
+import gof.strategy.domain.campaign.Campaign;
+import gof.strategy.repository.CampaignRepository;
+import gof.strategy.repository.ScheduledEventRepository;
+import gof.strategy.utils.Constants;
+import org.apache.commons.lang3.tuple.Pair;
+
+public class CampaignBannerUpdateStrategy extends AbstractEventStrategy {
+    private final CampaignRepository campaignRepository;
+    private final ScheduledEventRepository scheduledEventRepository;
+
+    public CampaignBannerUpdateStrategy(CampaignRepository campaignRepository, ScheduledEventRepository scheduledEventRepository) {
+        this.campaignRepository = campaignRepository;
+        this.scheduledEventRepository = scheduledEventRepository;
+    }
+
+    @Override
+    public ScheduledEvent process(ScheduledEvent event) {
+        super.process(event);
```

Comment:
This also should be in the `try-catch-block` because it can throw an exception. Also, as the result of the `super.process` the `event`'s status can be set to `FAILED` or `TERMINATED` so further processing does not make sense.